local-rendering.md

```diff
@@ -1 +1,36 @@
-# Serving the site locally
\ No newline at end of file
+# Serving the site locally
+
+## 1. Set the path of your content folder
```

Comment:
I think this should not be the first step. Instead making sure the user has npm installed or otherwise installing npm will be the first step. Setting the content folder path can come after installing npm and installing the dependencies with npm. So I think it would work better as step 3

Reply:
I agree. Thanks. One minor thing: `npm install` requires that a proper `CONTENT_PATH` is set as it also reads the `config.json`. So,  the order would be:
1. Install npm
2. Set the path
3. Setup npm
4. Start the development server

Reply:
Done.